app/config/locale/translations/en.json

```diff
@@ -3,12 +3,18 @@
     "settings.locale": "en",
     "settings.direction": "ltr",
     "emails.sender": "%s Team",
-    "emails.verification.subject": "Account Verification",
-    "emails.verification.hello": "Hey {{name}}",
-    "emails.verification.body": "Follow this link to verify your email address.",
-    "emails.verification.footer": "If you didn’t ask to verify this address, you can ignore this message.",
-    "emails.verification.thanks": "Thanks",
-    "emails.verification.signature": "{{project}} team",
+    "emails.verificationUrl.subject": "Account Verification",
+    "emails.verificationUrl.hello": "Hey {{name}}",
+    "emails.verificationUrl.body": "Follow this link to verify your email address.",
+    "emails.verificationUrl.footer": "If you didn’t ask to verify this address, you can ignore this message.",
+    "emails.verificationUrl.thanks": "Thanks",
+    "emails.verificationUrl.signature": "{{project}} team",
+    "emails.verificationCode.subject": "Account Verification",
+    "emails.verificationCode.hello": "Hey {{name}}",
+    "emails.verificationCode.body": "Please use the following code to verify your email address. Note that this code is only valid for 15 minutes.",
+    "emails.verificationCode.footer": "If you didn’t ask to verify this address, you can ignore this message.",
+    "emails.verificationCode.thanks": "Thanks",
+    "emails.verificationCode.signature": "{{project}} team",
```

Comment:
Lots of these are duplicate, what do you think about keeping ones that are as `"emails.verification.subject"` and specifying per type as `"emails.verification.code.body"` for example? Then we could fetch them without having to check the type like `"emails.verification.{$type}.body"`